hierarchicalforecast/core.py

```diff
@@ -45,9 +45,14 @@ def _reverse_engineer_sigmah(Y_hat_df, y_hat, model_name):
     In the future, we might deprecate this function in favor of a 
     direct usage of an estimated $\hat{sigma}_{h}$
     """
-    drop_cols = ['ds', 'y'] if 'y' in Y_hat_df.columns else ['ds']
+
+    drop_cols = ['ds']
+    if 'y' in Y_hat_df.columns:
+        drop_cols.append('y')
+    if model_name+'-median' in Y_hat_df.columns:
+        drop_cols.append(model_name+'-median')
     model_names = Y_hat_df.drop(columns=drop_cols, axis=1).columns.to_list()
-    pi_model_names = [name for name in model_names if ('-lo' in name or '-hi' in name)]
+    pi_model_names = [name for name in model_names if ('-lo' in name or '-hi' in name or '-median' in name)]
```

Comment:
Is this '-median' needed conditional on dropping it from model_names?

Reply:
You are right it is not needed, removed